File: Code coach/Easy/Jungle camping/jungle_camping.py
noiseToAnimal = {
	"Rawr": "Tiger",
	"Chirp": "Bird",
	"Ssss": "Snake",
	"Grr": "Lion",
}

noisesSplitted = input().split()
animals = ' '.join(map(lambda noise: noiseToAnimal[noise], noisesSplitted))
print(animals)

File: Code coach/Easy/Jungle camping/jungle_camping.rb
noiseToAnimal = {
	"Rawr" => "Tiger",
	"Chirp" => "Bird",
	"Ssss" => "Snake",
	"Grr" => "Lion",
}

noisesSplitted = gets.chomp.split(" ")
animals = noisesSplitted.map { |noise| noiseToAnimal[noise] }.join(" ")
puts animals
